npc/re/merchants/shops.txt

```diff
@@ -91,6 +91,25 @@ malangdo,232,163,5	shop	Tool Dealer Myo	560,611:-1,1750:-1,645:-1,656:-1,601:-1,
 mora,100,118,4	shop	Tool Dealer#mora	517,11522:-1,11523:-1,11524:-1,602:-1,601:-1,611:-1,1773:-1,1774:-1
 mora,138,110,4	shop	Mora Chef	518,11519:-1
 
+//=======================================================
+// Morroc
+//=======================================================
+morocc,156,76,4	shop	Butcher#moc1	58,517:-1
```

Comment:
Coordinates are wrong:
`trader "morocc" "Butcher" 4_M_03 157 72 6 0 0`

Reply:
according kRO navi_npc_krpri.lub, it's correct

Reply:
![imagen](https://user-images.githubusercontent.com/4557962/37289123-3f19ff16-2609-11e8-8a5d-0368d2bac6a3.png)
Matches my coordinates on aegis15.2, but it's completely absent from iRO, along with the tool dealer on the right

Reply:
iRO's content sometime unreliable,
they got many broken and missing updates

Reply:
I know, but client files aren't the most reliable source either (there's a lot of wrong skill descriptions, even in korean).

Reply:
```
trader "morocc" "고기상인" 1_M_MERCHANT 156 76 4 0 0 // 52311
	sellitem Meat // 50
```